src/v/kafka/server/handlers/list_partition_reassignments.cc

```diff
@@ -129,6 +129,7 @@ ss::future<response_ptr> list_partition_reassignments_handler::handle(
 
     // If topics is undefined, get all active reassignments
     if (!request.data.topics.has_value()) {
+        vlog(klog.debug, "Request to list all active reassignments");
```

Comment:
I recognize this is a commit for a 1 line change; however, there are currently issues in the ListPartitionReassignments API. So I made this commit as a placeholder for when I add those fixes in.

Reply:
> I recognize this is a commit for a 1 line change; however, there are currently issues in the ListPartitionReassignments API. So I made this commit as a placeholder for when I add those fixes in.

are you saying this PR isn't ready for review? not sure exactly what this comment means.

Reply:
> > I recognize this is a commit for a 1 line change; however, there are currently issues in the ListPartitionReassignments API. So I made this commit as a placeholder for when I add those fixes in.
> 
> are you saying this PR isn't ready for review? not sure exactly what this comment means.

I'm saying that I need to include fixes to ListPartitions to address this problem

>     * [x]  Add DT test: Put a partition twice in the same AlterPartitionReassignments request. First one should pass, second one should fail because the first one is still updating.

But we can start the review process on the things I have done already. The task list is simply pointing to things I have yet to address

Reply:
Addressed the problem.